roles/mediainfo/tasks/main.yml

```
@@ -1,12 +1,16 @@
 ---
 # tasks file for mediainfo
-- name: medianinfo | Install mediaserver repository
+- name: mediainfo | download mediainfo repo config
```

I'm confused by `config` here - isn't this the full mediainfo repo?
```suggestion
- name: mediainfo | download mediainfo repo
```